.github/workflows/github-actions-tests.yml

```diff
@@ -0,0 +1,37 @@
+# The name of the action
+name: Django Tests
+# When the action is triggered
+on:
+  push:
+    branches:
+      - master
```

Comment:
this should say `main` at the very least. Try to figure out if we can enable for all branches.